src/components/pages/404.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const NotFoundPage = () => {
+	return (
+		<div>
+			<h3>404 Not Found</h3>
+		</div>
+	);
+};
+
+export default NotFoundPage;
```

Comment:
we should also have a snapshot test for this page :) Check out `crcf` to automate that kinda thing, I think I mention it in the readme